static/js/pages/applications/ReviewDetailPage_test.js

```diff
@@ -0,0 +1,224 @@
+/* global SETTINGS: false */
+import { assert } from "chai"
+import sinon from "sinon"
+
+import IntegrationTestHelper from "../../util/integration_test_helper"
+import { makeUser } from "../../factories/user"
+import {
+  makeApplicationDetail,
+  makeSubmissionReview
+} from "../../factories/application"
+import {
+  REVIEW_STATUS_APPROVED,
+  REVIEW_STATUS_PENDING,
+  REVIEW_STATUS_REJECTED
+} from "../../constants"
+import { ReviewDetailPage } from "./ReviewDetailPage"
+
+import { shouldIf } from "../../lib/test_utils"
+import { getFilenameFromPath, isNilOrBlank } from "../../util/util"
+import wait from "waait"
+
+describe("ReviewDetailPage", () => {
+  let helper,
+    fakeUser,
+    renderPage,
+    fakeApplication,
+    fakeApplicationDetail,
+    fakeApplicationSubmission,
+    fakeSubmissionReview
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper()
+    fakeSubmissionReview = makeSubmissionReview()
+    fakeSubmissionReview.review_status = REVIEW_STATUS_PENDING
+    fakeApplication = fakeSubmissionReview.bootcamp_application
+    fakeUser = fakeSubmissionReview.learner
+    fakeApplicationSubmission = {
+      id:                      fakeSubmissionReview.id,
+      run_application_step_id: fakeSubmissionReview.run_application_step_id,
+      submitted_date:          fakeSubmissionReview.submitted_date,
+      review_status:           fakeSubmissionReview.review_status,
+      review_status_date:      fakeSubmissionReview.review_status_date
+    }
+    fakeApplicationDetail = makeApplicationDetail()
+    fakeApplicationDetail.id = fakeApplication.id
+    fakeApplicationDetail.bootcamp_run_id = fakeApplication.bootcamp_run_id
+    fakeApplicationDetail.submissions = [fakeApplicationSubmission]
+
+    helper.handleRequestStub
+      .withArgs(
+        `/api/applications/${fakeSubmissionReview.bootcamp_application.id}/`
+      )
+      .returns({
+        status: 200,
+        body:   fakeApplicationDetail
+      })
+
+    helper.handleRequestStub
+      .withArgs(`/api/submissions/${fakeSubmissionReview.id}/`)
+      .returns({
+        status: 200,
+        body:   fakeSubmissionReview
+      })
+
+    renderPage = helper.configureReduxQueryRenderer(
+      ReviewDetailPage,
+      {
+        match: {
+          params: {
+            id: fakeSubmissionReview.id
+          }
+        }
+      },
+      {
+        entities: {
+          currentUser: makeUser()
+        }
+      }
+    )
+  })
+
+  afterEach(() => {
+    helper.cleanup()
+  })
+
+  it("renders the user's details", async () => {
+    const { wrapper } = await renderPage()
+    assert.isTrue(wrapper.find("UserDetails").exists())
+  })
+
+  //
+  ;["http://resume.jpg", null].forEach(resumeFile => {
+    it(`${shouldIf(
+      !isNilOrBlank(resumeFile)
```

Comment:
Pretty sure this and several descriptions below can be rewritten as `${shouldIf(!!resumeFile)}`